Scc 3608/romcom 2.0

charmingduchess opened the pull request:
- remove deliverableTo from M2 items
 - add deliverableToResolution to all locations.csv entries
 - update serializer to accomodate new property
 
I ran validate changes locally to ensure nothing crazy was happening, but the output is too large because I updated all of the entries with deliverableToResolution data. Maybe I should remove the sierra-location data and we can default to sierra-location for entries with a blank deliverableToResolution...

nonword reviewed (commented):
> Maybe I should remove the sierra-location data and we can default to sierra-location for entries with a blank deliverableToResolution...

I think that would be fine. Deliverable-to resolution defaulting to a sierra-location lookup seems like an intuitive default to me.

charmingduchess commented on README.md:
```diff
@@ -84,3 +84,12 @@ git push --tags -f
+
+
+## Locations and deliverableToResolution
+There are three different sources of truth for determining deliverableTo: Sierra, Recap, and the M2 ILS. 
```
ok! what is GFA then?

charmingduchess replied:
inventory system?

nonword replied:
Yeah that works.

nonword replied:
Seems to have reverted to ILS?

charmingduchess commented:
@nonword here's validate-changes after removing sierra-locations as explicit value. there are also somewhat extraneous commits here due to some weirdness regarding use of both ’ and ' in labels... perhaps we can standardize those in a different pr.

Comparing locations in SCC-3608/romcom-2.0 to master
Keys Added: 0
Keys Deleted: 0
Keys Altered: 81

ALTERED KEY: nyplLocation:rccf9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0f
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrjlf
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rc2ma
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0y
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:map92
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpm9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmb2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0l
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:mal99
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcce2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpt2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcce8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccd2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpd8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmj2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0i
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rc
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0h
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccb2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpd9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcca9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccf2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:mas92
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:mal92
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmo2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmf2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcph9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccf8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcxx2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcce9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmb9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rc2sl
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmf8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmb8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmi2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmg8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpm2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:mal98
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpf2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rc2mj
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0t
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpt8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpt9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpd2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmf9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpr2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj01
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:mab88
Added Item: root['nypl:deliverableTo']

ALTERED KEY: nyplLocation:xrjr
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0v
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:mag92
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:mai82
Attribute: root['nypl:deliverableTo']
Old Value: {'@id': 'nyplLocation:mai'}
New Value: [{'@id': 'nyplLocation:mab'}, {'@id': 'nyplLocation:mag'}, {'@id': 'nyplLocation:map'}, {'@id': 'nyplLocation:maf'}, {'@id': 'nyplLocation:mal'}, {'@id': 'nyplLocation:mai'}, {'@id': 'nyplLocation:mala'}, {'@id': 'nyplLocation:malc'}, {'@id': 'nyplLocation:maln'}, {'@id': 'nyplLocation:malw'}, {'@id': 'nyplLocation:mal17'}]

ALTERED KEY: nyplLocation:rc2cf
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpm8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmg9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcpr9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccb8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0n
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrj0a
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmp2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:maf92
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcph8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcml8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:maf99
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccd9
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcml2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcx28
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccd8
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:map98
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcms2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:xrjln
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:maf98
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmq2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmg2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcsl2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcma2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rc2cm
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcph2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rcmr2
Added Item: root['nypl:deliverableToResolution']

ALTERED KEY: nyplLocation:rccb9
Added Item: root['nypl:deliverableToResolution']

nonword commented on vocabularies/scripts/__pycache__/m2Utils.cpython-39.pyc:
Can you remove this?
